warehouse/search/queries.py

```diff
@@ -10,26 +10,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SEARCH_FIELDS = [
-    "author",
-    "author_email",
-    "description",
-    "download_url",
-    "home_page",
-    "keywords",
-    "license",
-    "maintainer",
-    "maintainer_email",
-    "normalized_name",
-    "platform",
-    "summary",
-]
 SEARCH_BOOSTS = {
+    "name.keyword": 20,
     "name": 10,
     "normalized_name": 10,
     "description": 5,
     "keywords": 5,
     "summary": 5,
+    "author": 1,
+    "author_email": 1,
+    "download_url": 1,
+    "home_page": 1,
+    "license": 1,
+    "maintainer": 1,
+    "maintainer_email": 1,
+    "platform": 1,
 }
```

Comment:
This is a small refactoring - I removed the duplicate definition of search fields. Now you only need to state search fields once and you must set a boost for it.